File: RentACarProject/RentACarProject/DTO/RentiranjeDTO.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using RentACarProject.Models;

namespace RentACarProject.DTO
{
    public class RentiranjeDTO
    {
        
        public decimal BrojDanaIzdavanja { get; set; }
        public DateTime DatumPocetkaIzdavanja { get; set; }
        public DateTime DatumKrajaIzdavanja { get; set; }
        public decimal UkupnaCenaRentiranja { get; set; }
        public string PlacenoR { get; set; } = null!;
        public DateTime? DatumPlacanja { get; set; }
        public decimal? PristupniKod { get; set; }
        public Guid KorisnikId { get; set; }
        public virtual Korisnik Korisnik { get; set; } = null!;
        public Guid ZaposleniId { get; set; }
        public virtual Korisnik Zaposleni { get; set; } = null!;
        public Guid AutomobilId { get; set; }
        public virtual Automobil Automobil { get; set; } = null!; 
    }
}

File: ERP-RentACar-FrontEnd-Project-IT60-2020/src/app/models/rentiranjeDTO.ts
import { Guid } from "guid-typescript";

export class RentiranjeDTO {
    brojDanaIzdavanja!:number;
    datumPocetkaIzdavanja!:Date;
    datumKrajaIzdavanja!:Date;
    ukupnaCenaRentiranja!:number;
    placenoR!:String;
    datumPlacanja?:Date;
    pristupniKod!:number;
    automobilId!:string;
    korisnikId!:string;
    zaposleniId!:string;
    stripeToken!:String;

    /*
    constructor(
        brojDanaIzdavanja:number,
        datumPocetkaIzdavanja:Date,
        datumKrajaIzdavanja:Date,
        ukupnaCenaRentiranja:number,
        placenoR:String,
        datumPlacanja:Date,
        pristupniKod:number,
        automobilId:string,
        korisnikId:Guid,
        zaposleniId:Guid,
        stripeToken:String
    ) {
        this.brojDanaIzdavanja = brojDanaIzdavanja;
        this.datumPocetkaIzdavanja = datumPocetkaIzdavanja;
        this.datumKrajaIzdavanja = datumKrajaIzdavanja;
        this.ukupnaCenaRentiranja = ukupnaCenaRentiranja;
        this.placenoR = placenoR;
        this.datumPlacanja = datumPlacanja;
        this.pristupniKod = pristupniKod;
        this.automobilId = automobilId;
        this.korisnikId = korisnikId;
        this.zaposleniId = zaposleniId;
        this.stripeToken = stripeToken;
    }
    */
}